### PR TITLE
fix: freeze @swc/core to 1.5.7

### PR DESCRIPTION
**Pull request by vscaiceanu-1a**
## Proposed change

Cherry pick from https://github.com/AmadeusITGroup/otter/pull/1831

### CODE REVIEW

**Review by mrednic-1A — APPROVED** [review]

File: packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/package.json.template [hunk 1]
```diff
@@ -83,7 +83,7 @@
     "@openapitools/openapi-generator-cli": "<%= versions['@openapitools/openapi-generator-cli'] %>",
     "@stylistic/eslint-plugin-ts": "<%= versions['@stylistic/eslint-plugin-ts'] %>",
     "@swc/cli": "^0.1.57",
-    "@swc/core": "^1.3.85",
+    "@swc/core": "1.5.7",
```
Comment (kpanot) [inline packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/package.json.template:86]: should be retrieve from package.json like the other dependencies

**Comment by vscaiceanu-1a** [comment]
> Not sure to understand, why is it a cherry-pick instead of a cascading?

The ITs were failing on the merge queue and this is blocking other PRs and the RC creation on main so a cherry-pick is faster.

File: packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/package.json.template (hunk 1 above)
Comment (vscaiceanu-1a) [reply packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/package.json.template:86]: It should be, if `packages/@ama-sdk/schematics/package.json` had this dep :)

Comment (vscaiceanu-1a) [reply packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/package.json.template:86]: https://github.com/AmadeusITGroup/otter/issues/1835

**Pull request merged**